core/domain/interaction_jobs_one_off_test.py

```diff
@@ -1356,3 +1356,112 @@ def test_no_action_is_performed_for_deleted_exploration(self):
                 interaction_jobs_one_off
                 .RuleInputToCustomizationArgsMappingOneOffJob)
         )
+
+
+class LogicProofInteractionOneOffJobTests(test_utils.GenericTestBase):
+
+    ALBERT_EMAIL = 'albert@example.com'
+    ALBERT_NAME = 'albert'
+
+    VALID_EXP_ID = 'exp_id0'
+    NEW_EXP_ID = 'exp_id1'
+    EXP_TITLE = 'title'
+
+    def setUp(self):
+        super(LogicProofInteractionOneOffJobTests, self).setUp()
+
+        # Setup user who will own the test explorations.
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
```

Comment:
Please avoid using real names in tests, use "anonymous" values like `self.USER_EMAIL = user@example.com`.

Reply:
Which did you see it in? I don't think that should be the convention, and don't remember seeing it used anywhere (we already have `self.USER_EMAIL`).

@vojtechjelinek WDYT?

Reply:
I think it is used in some places (https://github.com/oppia/oppia/search?q=ALBERT_NAME), but yeah let's use more generic variables like USER.